{{ cookiecutter.project_name }}/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="{{ cookiecutter.package_name }}",
+    author="{{ cookiecutter.author_name }}",
+    author_email="{{ cookiecutter.author_email }}",
+    packages=find_packages(),
+    install_requires=[],
```

Comment:
This should probably be connected to the `requirements.txt` file that currently lives in ` {{ cookiecutter.project_name }}/docker/`.

Reply:
+1

Reply:
Ya, so I think we definitely need documentation in this PR around this. Using `install_requires` vs `requirements.txt` depends on what you're doing.

1. building a package: use `install_requires` and don't pin versions, i.e. do `Click>=1.0.0`
2. building an app as a docker container: use `requirements.txt` and pin versions

Thoughts on just documenting this?

Ref: https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
I think i'm ok with leaving it like this (given your comment above). As I see it, not everyone will make a package out of this repo. It's an option -- but you have to customize as you see fit when you use this option.